website/docs/releases/2025/v2025.6.md

```diff
@@ -0,0 +1,82 @@
+
+- **mTLS stage**: :ak-enterprise The Mutual TLS stage enables authentik to use client certificates to enroll and authenticate users. These certificates can be local to the device or available via PIV Smart Cards, Yubikeys, etc. For environments where certificates are already rolled out, this can make authentication a lot more seamless. Refer to our [technical documentation](../add-secure-apps/flows-stages/stages/mtls/) for more information.
+- **Email verification compatibility with link scanners**: We have improved compatibility for environments with automated scanning software that inadvertently invalidated one-time links sent by authentik.
+- **LDAP source sync forward deletions**: With this option enabled, users who were created in authentik via LDAP sources will also be removed from authentik if they are deleted from the LDAP source. For more information, please refer to our [LDAP source documentation](../users-sources/sources/protocols/ldap/).
+- **Provider sync performance**: We have implemented parallel scheduling for outgoing syncs to provide faster synchronization.
+- **Branding**: Custom branding should now be more consistent on initial load, without flickering.
+
+## New integration guides
+
+An integration is how authentik connects to third-party applications, directories, and other identity providers. The following integration guides were recently added.
+
+- [Pangolin](../../../integrations/services/pangolin/)
+- [Stripe](../../../integrations/services/stripe/)
+- [FileRise](../../../integrations/services/filerise/)
+- [Push Security](../../../integrations/services/push-security/)
+- [Atlassian Cloud (Jira, Confluence, etc)](../../../integrations/services/atlassian/)
+- [Coder](../../../integrations/services/coder/)
+- [YouTrack](../../../integrations/services/youtrack/)
+- [Komodo](../../../integrations/services/komodo/)
+
+## Upgrading
+
+This release does not introduce any new requirements. You can follow the upgrade instructions below; for more detailed information about upgrading authentik, refer to our [Upgrade documentation](../../install-config/upgrade.mdx).
+
+:::warning
+When you upgrade, be aware that the version of the authentik instance and of any outposts must be the same. We recommended that you always upgrade any outposts at the same time you upgrade your authentik instance.
+:::
+
+### Docker Compose
+
+To upgrade, download the new docker-compose file and update the Docker stack with the new version, using these commands:
+
+```shell
+wget -O docker-compose.yml https://goauthentik.io/version/2025.6/docker-compose.yml
+docker compose up -d
+```
+
+The `-O` flag retains the downloaded file's name, overwriting any existing local file with the same name.
+
+### Kubernetes
+
+Upgrade the Helm Chart to the new version, using the following commands:
+
+```shell
+helm repo update
+helm upgrade authentik authentik/authentik -f values.yaml --version ^2025.6
+```
+
+## Minor changes/fixes
+
+TODO
```

Comment:
Is this new? 

At any rate, since I got so much pushback about adding an entry for "Improved RAC docs" up under New Feautres and improvements, I guess we can add it here.